packages/web-service/src/pages/eligibility/__tests__/__snapshots__/eligibility.spec.js.snap

```diff
@@ -173,8 +173,8 @@ exports[`the eligibility pages the eligibility page template Matches the snapsho
                 
 
     
-        <h2 class=\\"govuk-heading-m\\">Save application</h2>
-        <p class=\\"govuk-body\\">To be able to save your application and return to it later, you need to sign into an existing account or create a new one.</p>
+        <p class=\\"govuk-body\\">To apply for this licence online, you must have Government Gateway sign in details.</p>
```

Comment:
additional false stop that is not on the ticket... 

  - paragraph 1:"To apply for this licence online, you must have Government Gateway sign in details”